### ISSUE TITLE
Doesn't seem to work in 0.208.0

### ISSUE THREAD

**Issue by Scratch2k** [harai/indent-guide-improved#13]
I put the repo in the packages folder (unzipped) and it appears in the Packages section within Atom but  the guides have not auto disabled and the indent guides do not appear, any ideas?

**Comment by ptrwllrd** [comment]
same here

**Comment by harai** [comment]
Disabling Shadow DOM seems to prevent this package from working.
Please check "Use Shadow DOM" in Preferences.

**Comment by MethodGrab** [comment]
I had this problem on Atom 0.208.0 as well (now 0.209.0, wow Atom updates a lot!).
For me, "Use Shadow DOM" was already enabled but the guides still don't show up! I tried both enabled and disabled and can't get the improved guides to appear.

**Comment by harai** [comment]
I confirmed the problem after upgrading to 0.208.0.

**Comment by harai** [comment]
Atom changes so fast that it is hard to follow.

**Comment by MethodGrab** [comment]
Thanks, it must make maintaining Atom packages quite annoying!

**Comment by Scratch2k** [comment]
1.3.0 tested in Atom 0.209.0 and working, thanks!
